src/core/reactor_backend.cc

```diff
@@ -1340,6 +1484,11 @@ class reactor_backend_uring final : public reactor_backend {
             case o::fdatasync:
                 ::io_uring_prep_fsync(sqe, req.fd(), IORING_FSYNC_DATASYNC);
                 break;
+#ifdef USE_PASSTHRU
+            case o::append:
+                prep_passthru_append(sqe, req.fd(), req.address(), req.size(), req.pos(), req.nsid(), req.block_size(), completion);
```

Comment:
nit: Passing `const internal::io_request&` would make this place much easier to read